sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/BatchScanExec.scala

```diff
@@ -18,7 +18,6 @@
 package org.apache.spark.sql.execution.datasources.v2
 
 import com.google.common.base.Objects
-
```

Comment:
We should keep this blank line.

Reply:
Done, this also fixes the checkstyle issue.